### ISSUE TITLE
autoHeight/autoHeightMax in Edge always shows scrollbar

### ISSUE THREAD

**Issue by magnetronnie** [malte-wessel/react-custom-scrollbars#113]
I have this setup:
```
<div className="inner-container">
	<Scrollbars autoHeight={true} autoHeightMax={200} style={{ width: "100%" }}>
		Content..
	</Scrollbars>
</div>
```

```
.inner-container {
    max-height: 200px;
}
```

The content is about 125px in height. In FF and Chrome it works fine, but in Edge I always get a vertical scrollbar and it's scrollable for 1 or 2 pixels. I have several 'widgets' with different content (ul, div, table) and the same thing happens everywhere, no matter the height and content.

Is this a bug or am I missing something?

**Comment by Soulfull** [comment]
I have the same problem too.  `scrollHeight` property returns incorrect value in Edge.

**Comment by ofermusaiwork** [comment]
try to add Math.ceil to the calculation of the scroll

in function getThumbVerticalHeight
change: `height = clientHeight / scrollHeight * trackHeight;`
to: `height = Math.ceil(clientHeight / scrollHeight * trackHeight);`

and also in the width in getThumbHorizontalWidth

**Comment by malte-wessel** [comment]
Fixed in v4.0.2
